dbdev/models.py

```diff
@@ -0,0 +1,45 @@
+import psycopg2
+
+from flask import abort,Response
+
+
+connect = psycopg2.connect("dbname='ridemyway' host='localhost' user='dario' password='riot'")
+curs = connect.cursor()
+
+
+
+#Class to add new users to the db
+class DbManager(object):
+      
+    #class methods
+    def __init__(self):
+        pass
+
+    def userRegDetails(self,email,username,password):
```

Comment:
It's not clear what this section of your code does. From my understanding, you are trying to register a user but the inputs of the function are not being validated. Have you checked what a constructor does?